cmd/tracee-ebpf/main.go

```diff
@@ -29,6 +30,28 @@ func init() {
 			},
 		)
 	}
+
+	// Set libbpfgo logging callbacks
+	libbpfgo.SetLoggerCbs(libbpfgo.Callbacks{
+		Log: func(libLevel int, msg string, keyValues ...interface{}) {
+			lvl := logger.ErrorLevel
+
+			switch libLevel {
+			case libbpfgo.LibbpfWarnLevel:
+				lvl = logger.WarnLevel
+			case libbpfgo.LibbpfInfoLevel:
+				lvl = logger.InfoLevel
+			case libbpfgo.LibbpfDebugLevel:
+				lvl = logger.DebugLevel
+			}
+
+			logger.Log(lvl, false, msg, keyValues...)
+		},
+		LogFilters: []func(libLevel int, msg string) bool{
+			libbpfgo.LogFilterLevel,
+			libbpfgo.LogFilterOutput,
```

Comment:
So you're planning on using the default LogFilter function, then making it generic in libbpfgo, then bumping libbpfgo again in Tracee (that time with a function filtering the output). Is that so ?

Reply:
If I got you right, it's not my intent. This PR only closes the cycle of the backlog in libbpfgo without changing behaviour of both projects.

The next step must be to remove the consumer logic from libbfpgo and place it, in our case, in tracee. See https://github.com/aquasecurity/libbpfgo/issues/293.

After this merge I'm opening this, in tracee, as well:

![image](https://user-images.githubusercontent.com/3372117/220199218-41c5b5dd-d9a1-4204-89d0-24b4a65798ff.png)

Reply:
Cool that is exactly what I meant =D. Sorry for the miss interpretation.